cpg_workflows/large_cohort/sample_qc.py

```diff
@@ -15,7 +15,27 @@
 from gnomad.sample_qc.pipeline import annotate_sex
 
 
+def can_reuse_dataset(path: str):
+    """
+    checks for existence of a Path
+    if the path is a MT or VDS, checks for the success file
+    """
+
+    if not path:
+        return False
+
+    path_as_path = to_path(path)
+
+    if path_as_path.suffix in ['.mt', '.ht']:
+        path_as_path /= '_SUCCESS'
+    if path_as_path.suffix in ['.vds']:
+        path_as_path /= 'variant_data/_SUCCESS'
+
+    return path_as_path.exists()
```

Comment:
this should be reused from the cpg_workflows module

Reply:
```
from cpg_workflows.utils import can_reuse
```

Reply:
can_reuse calls exists, exists calls exists_not_cached, exists_not_cached does this `_SUCCESS` check
https://github.com/populationgenomics/production-pipelines/blob/d37ca03a06bb9b8c0df22ea8ac13d3ed1d5de797/cpg_workflows/utils.py#L181